RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -810,6 +810,11 @@ boolean L12_filthworms()
 	{
 		auto_log_info("Ed will steal stench glands using [Lash of the Cobra]");
 	}
+	// or if we're about to yellow ray
+	else if(auto_haveYellowRayAvailable())
+	{
+		auto_log_info("We're going to yellow ray the stench glands.");
+	}
```

Comment:
I would move this below all the forced pickpockets for consistency (so it is above the else clause) since it's the failover situation when all of those aren't applicable.